x/gov/testutil/expected_keepers.go

```diff
@@ -33,3 +33,7 @@ type StakingKeeper interface {
 	BondDenom(ctx sdk.Context) string
 	TokensFromConsensusPower(ctx sdk.Context, power int64) math.Int
 }
+
+type DistributionKeeper interface {
+	types.DistributionKeeper
```

Comment:
why do we need the entire interface? We should only describe what we need, i.e. `FundCommunityPool`.